### DISCUSSION TITLE
license 能否同时绑定多个机器？

### DISCUSSION THREAD

**Discussion by cspshare** [dashingsoft/pyarmor#2010] [General]
license 能否同时绑定多个机器，以方便在集群环境下使用。

**Comment by jondy** [comment]
可以的，使用选项 `--bind` 多次，可以绑定多个机器

**Reply by cspshare** [reply]
发现7.6的版本不能多次使用--bind，最终绑定的是最后一个bind。之前买了7.6的licenses，新版本是不是需要再买一个licenses?

**Comment by jondy** [comment]
请参阅文档中升级说明，或者 issue #1958

**Reply by aijcode** [reply]
谢谢，已经免费升级至8，后面再看要不要升级至专家版
